_posts/2019/2019-09-18-you.md

```diff
@@ -0,0 +1,111 @@
+
+Coming to the office last Thursday was a bit different than usual.
+Many people were already standing together, talking and laughing in the hallway.
+This first impression was already promising a welcoming and more casual atmosphere.
+Quickly grabbing a snack, it was time to join the others.
+
+### "Welcome to the ePages YOU!"
+
+Wilfried Beeck, CEO of ePages, gave us an update on where ePages is heading to in the future.
+He also presented how ePages is working across the world with existing and future partners.
+
+{% twitter https://twitter.com/epagesdevs/status/1172044512421658630 %}
+
+Afterwards, Tatu Wikman, CTO of the [Vilkas Group](https://www.vilkasgroup.com/), who has been an ePages’ technology partner for many years now, joined us and talked about how Vilkas works and what they offer to achieve the best possible ePages shop solution.
+Vilkas is known for its high rate of customer satisfaction.
+Tatu emphasized that the key to keep customers satisfied is to support them in whatever they are doing.
+
+### It’s people-BINGO-time!
+
+To raise conversation beyond the familiar office doors, an ice breaker was organized.
+This year it was bingo.
+We each received a table with certain characteristics.
+The goal was to find people who were not working on your office floor and who match those characteristics, so that eventually you’d get a Bingo by putting their names in the table.
+It was quite fun and a good opportunity to get to know new people from the company.
+So at the end of the day, even if you know nothing else about that person, you’ll know if he or she owns a Madonna CD.
+
+### Know your customer
+
+After a short break we had a panel discussion with four merchants on the agenda.
+This was probably our favorite part because we got first-hand feedback in an open discussion from merchants directly using our software on a daily basis.
+They addressed what they already like about the current software but also which changes and features they would like to see in the future.
+
+{% twitter https://twitter.com/epagesdevs/status/1172078677452120064 %}
+
+It was great to see what we already achieved in terms of our online shop software but also to recognize what still needs to be done in order to meet the merchants needs.
+Thanks again to Kati Decker from superstreusel.de, Imme Hansemann from hansemann.de, Michael Mommsen from nutringo.de and Heinrich Dumrath from chronobalance.de for your time and opinion.
+
+### Work it
+
+After a little lunch break, we continued with several workshops for which we could sign up for during the last weeks.
+They all covered different topics, such as usability testing, customer micro-moments and journeys, and a perspective change game.
+
+{% image_custom image="/assets/img/pages/blog/images/blog-you19-2.jpg" width="50" lightbox %}
+
+After the end of the workshops, the different groups made their way back to the presentation room where each group presented what they had worked on for the past two hours.
```

Comment:
After the workshops, the different groups made their way back to the presentation room where each group presented what they had worked on for the past two hours.